src/main/java/karuna/karuna_backend/Models/Content.java

```diff
@@ -0,0 +1,28 @@
+package karuna.karuna_backend.Models;
+
+import jakarta.annotation.Nullable;
+import jakarta.persistence.*;
```

Comment:
Do not use imports with *.
https://medium.com/@aayushsharda/why-is-it-better-to-avoid-in-import-statements-in-java-ff9ceaca8aeb